src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf?) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf) format("ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc) format("ttc");
+}
+
+* {
+  font-family: "Avenir-Book", sans-serif;
+}
```

Comment:
ok for now, but do not use this selector for setting the font. `*` can only be used for box-sizing. set font for `html`